README.adoc

```diff
@@ -1,102 +1,405 @@
+:uri-mozilla-addon-image: https://blog.mozilla.org/addons/files/2020/04/get-the-addon-fx-apr-2020.svg
+:uri-opera-addon-image: https://dev.opera.com/extensions/branding-guidelines/addons_206x58_en@2x.png
+:uri-edge-webstore-image: https://raw.githubusercontent.com/asciidoctor/asciidoctor-browser-extension/main/promotional/edgewebstore-badge.png
+:uri-asciidoctor-image: https://avatars.githubusercontent.com/u/3137042?s=200&v=4
+// IMPORTANT - Change the branch in the URIs below - This is for testing only
+:uri-ext-active-image: https://raw.githubusercontent.com/sturtison/asciidoctor-browser-extension/docs/issue-648-refresh-docs-draft/promotional/ext-active.png
+:uri-ext-inactive-image: https://raw.githubusercontent.com/sturtison/asciidoctor-browser-extension/docs/issue-648-refresh-docs-draft/promotional/ext-inactive.png
+
+
+image::{uri-asciidoctor-image}[Asciidoctor icon,role="related thumb left",height="50px",width="50px"]
+
+Enjoy AsciiDoc everywhere your browser goes with the {project-name}.
+
+The {project-name} uses {uri-asciidoctor-js-repo}[Asciidoctor.js] 
+to preview {uri-asciidoc-docs}[AsciiDoc] files (.ad, .adoc, .asc, .asciidoc, and optionally .txt)
+as HTML inside Chrome, Brave, Firefox, Opera, Microsoft Edge and other Chromium browsers.
+
+Learn more about AsciiDoc and use its online demo interactive editor at https://asciidoc.org
 
-This project uses https://github.com/asciidoctor/asciidoctor.js[Asciidoctor.js] to preview AsciiDoc as HTML inside Chrome (or Chromium), Firefox and Opera!
+<<usage>> - <<get-started>> - {uri-project-repo}/issues[Submit an issue] - <<example>> - <<new-to-asciidoc>> - <<options>>
 
-image:https://raw.githubusercontent.com/asciidoctor/asciidoctor-browser-extension/main/promotional/chromewebstore-badge-wborder-496x150.png[link={uri-chrome-webstore},height="64px",width="212px"]
-image:https://blog.mozilla.org/addons/files/2020/04/get-the-addon-fx-apr-2020.svg[link={uri-mozilla-addon},height="64px",width="184px"]
-image:https://dev.opera.com/extensions/branding-guidelines/addons_206x58_en@2x.png[link={uri-opera-addon},height="64px",width="227px"]
-image:https://raw.githubusercontent.com/asciidoctor/asciidoctor-browser-extension/main/promotional/edgewebstore-badge.png[link={uri-edge-webstore},height="64px",width="177px"]
+'''
 
 *Project health:* image:https://github.com/asciidoctor/asciidoctor-browser-extension/workflows/CI/badge.svg[Build Status, link="https://github.com/asciidoctor/asciidoctor-browser-extension/actions?query=workflow%3ACI"]
 
+'''
+
+Follow the image links below to install the extension direct from your browser's webstore or search your browser's webstore for the Asciidoctor.js Live Preview extension.
+
+[%autowidth,cols="a,a,a,a"]
+|===
+|.Chrome
+image::{uri-chrome-webstore-image}["Chrome, Chromium or Brave",link={uri-chrome-webstore},height="64px",width="212px"]
```

Comment:
> ,height="64px",width="212px"

height and width should not have units. Also, they can be written using positional attributes.

```
image::{uri-chrome-webstore-image}["Chrome, Chromium, or Brave",212,64,link={uri-chrome-webstore}]
```

FYI: A block attribute value only has to be enclosed in quotes if it contains a comma.